### ISSUE TITLE
crash when loading schematic

### ISSUE THREAD

**Issue by zeavic** [Lunatrius/Schematica#369]
my game crashes when i click done after picking a schematic.
1.12.2

crash report
```
---- Minecraft Crash Report ----

WARNING: coremods are present:
  CreativePatchingLoader (CreativeCore_v1.9.25_mc1.12.2.jar)
  ItemPatchingLoader (ItemPhysic_Full_1.4.13_mc1.12.2.jar)
  ObfuscatePlugin (Obfuscate-Mod-1.12.2.jar)
Contact their authors BEFORE contacting forge

// Shall we play a game?

Time: 7/30/18 6:04 PM
Description: Unexpected error

java.lang.NullPointerException: The validated object is null
	at org.apache.commons.lang3.Validate.notNull(Validate.java:225)
	at org.apache.commons.lang3.Validate.notNull(Validate.java:206)
	at stellarium.render.stellars.layer.StellarLayerModel.onStellarTick(StellarLayerModel.java:78)
	at stellarium.render.stellars.phased.StellarRenderModel.onTick(StellarRenderModel.java:64)
	at stellarium.render.stellars.StellarModel.onTick(StellarModel.java:41)
	at stellarium.render.sky.SkyModel.onTick(SkyModel.java:64)
	at stellarium.ClientProxy.updateTick(ClientProxy.java:179)
	at stellarium.StellarTickHandler.tickStart(StellarTickHandler.java:41)
	at net.minecraftforge.fml.common.eventhandler.ASMEventHandler_92_StellarTickHandler_tickStart_ClientTickEvent.invoke(.dynamic)
	at net.minecraftforge.fml.common.eventhandler.ASMEventHandler.invoke(ASMEventHandler.java:90)
	at net.minecraftforge.fml.common.eventhandler.EventBus.post(EventBus.java:182)
	at net.minecraftforge.fml.common.FMLCommonHandler.onPreClientTick(FMLCommonHandler.java:343)
	at net.minecraft.client.Minecraft.func_71407_l(Minecraft.java:1706)
	at net.minecraft.client.Minecraft.func_71411_J(Minecraft.java:1096)
	at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:397)
	at net.minecraft.client.main.Main.main(SourceFile:123)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at net.minecraft.launchwrapper.Launch.launch(Launch.java:135)
	at net.minecraft.launchwrapper.Launch.main(Launch.java:28)


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Thread: Client thread
Stacktrace:
	at org.apache.commons.lang3.Validate.notNull(Validate.java:225)
	at org.apache.commons.lang3.Validate.notNull(Validate.java:206)
	at stellarium.render.stellars.layer.StellarLayerModel.onStellarTick(StellarLayerModel.java:78)
	at stellarium.render.stellars.phased.StellarRenderModel.onTick(StellarRenderModel.java:64)
	at stellarium.render.stellars.StellarModel.onTick(StellarModel.java:41)
	at stellarium.render.sky.SkyModel.onTick(SkyModel.java:64)
	at stellarium.ClientProxy.updateTick(ClientProxy.java:179)
	at stellarium.StellarTickHandler.tickStart(StellarTickHandler.java:41)
	at net.minecraftforge.fml.common.eventhandler.ASMEventHandler_92_StellarTickHandler_tickStart_ClientTickEvent.invoke(.dynamic)
	at net.minecraftforge.fml.common.eventhandler.ASMEventHandler.invoke(ASMEventHandler.java:90)
	at net.minecraftforge.fml.common.eventhandler.EventBus.post(EventBus.java:182)
	at net.minecraftforge.fml.common.FMLCommonHandler.onPreClientTick(FMLCommonHandler.java:343)

-- Affected level --
Details:
	Level name: MpServer
	All players: 1 total; [EntityPlayerSP['Involen'/1459, l='MpServer', x=-593.49, y=103.00, z=163.95]]
	Chunk stats: MultiplayerChunkCache: 591, 591
	Level seed: 0
	Level generator: ID 01 - flat, ver 0. Features enabled: false
	Level generator options: 
	Level spawn location: World: (-384,4,481), Chunk: (at 0,0,1 in -24,30; contains blocks -384,0,480 to -369,255,495), Region: (-1,0; contains chunks -32,0 to -1,31, blocks -512,0,0 to -1,255,511)
	Level time: 5008 game time, 5008 day time
	Level dimension: 0
	Level storage version: 0x00000 - Unknown?
	Level weather: Rain time: 0 (now: false), thunder time: 0 (now: false)
	Level game mode: Game mode: creative (ID 1). Hardcore: false. Cheats: false
	Forced entities: 4 total; [EntityChicken['Chicken'/1505, l='MpServer', x=-591.50, y=103.00, z=243.50], EntityPlayerSP['Involen'/1459, l='MpServer', x=-593.49, y=103.00, z=163.95], EntityCow['Cow'/1518, l='MpServer', x=-518.07, y=103.00, z=119.06], EntityCow['Cow'/1519, l='MpServer', x=-515.50, y=103.00, z=126.50]]
	Retry entities: 0 total; []
	Server brand: fml,forge
	Server type: Integrated singleplayer server
Stacktrace:
	at net.minecraft.client.multiplayer.WorldClient.func_72914_a(WorldClient.java:420)
	at net.minecraft.client.Minecraft.func_71396_d(Minecraft.java:2738)
	at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:426)
	at net.minecraft.client.main.Main.main(SourceFile:123)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at net.minecraft.launchwrapper.Launch.launch(Launch.java:135)
	at net.minecraft.launchwrapper.Launch.main(Launch.java:28)

-- System Details --
Details:
	Minecraft Version: 1.12.2
	Operating System: Windows 10 (amd64) version 10.0
	Java Version: 1.8.0_51, Oracle Corporation
	Java VM Version: Java HotSpot(TM) 64-Bit Server VM (mixed mode), Oracle Corporation
	Memory: 252573336 bytes (240 MB) / 1073741824 bytes (1024 MB) up to 1073741824 bytes (1024 MB)
	JVM Flags: 8 total; -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xmx1G -XX:+UnlockExperimentalVMOptions -XX:+UseG1GC -XX:G1NewSizePercent=20 -XX:G1ReservePercent=20 -XX:MaxGCPauseMillis=50 -XX:G1HeapRegionSize=16M
	IntCache: cache: 0, tcache: 0, allocated: 0, tallocated: 0
	FML: MCP 9.42 Powered by Forge 14.23.4.2705 20 mods loaded, 20 mods active
	States: 'U' = Unloaded 'L' = Loaded 'C' = Constructed 'H' = Pre-initialized 'I' = Initialized 'J' = Post-initialized 'A' = Available 'D' = Disabled 'E' = Errored

	| State          | ID                | Version        | Source                                      | Signature                                |
	|:-------------- |:----------------- |:-------------- |:------------------------------------------- |:---------------------------------------- |
	| UCHIJAAAAAAAAA | minecraft         | 1.12.2         | minecraft.jar                               | None                                     |
	| UCHIJAAAAAAAAA | mcp               | 9.42           | minecraft.jar                               | None                                     |
	| UCHIJAAAAAAAAA | FML               | 8.0.99.99      | forge-1.12.2-14.23.4.2705.jar               | e3c3d50c7c986df74c645c0ac54639741c90a557 |
	| UCHIJAAAAAAAAA | forge             | 14.23.4.2705   | forge-1.12.2-14.23.4.2705.jar               | e3c3d50c7c986df74c645c0ac54639741c90a557 |
	| UCHIJAAAAAAAAA | creativecoredummy | 1.0.0          | minecraft.jar                               | None                                     |
	| UCHIJAAAAAAAAA | itemphysic        | 1.4.0          | minecraft.jar                               | None                                     |
	| UCHIJAAAAAAAAA | obfuscate         | 0.2.4          | minecraft.jar                               | None                                     |
	| UCHIJAAAAAAAAA | bibliocraft       | 2.4.5          | BiblioCraft[v2.4.5][MC1.12.2].jar           | None                                     |
	| UCHIJAAAAAAAAA | cdm               | 0.4.1          | cdm-0.4.1-mc1.12.2.jar                      | None                                     |
	| UCHIJAAAAAAAAA | cfm               | 5.4.1          | cfm-5.4.1-mc1.12.2.jar                      | None                                     |
	| UCHIJAAAAAAAAA | creativecore      | 1.9.9          | CreativeCore_v1.9.25_mc1.12.2.jar           | None                                     |
	| UCHIJAAAAAAAAA | fw                | 1.6.0          | FullscreenWindowed-1.12-1.6.0.jar           | None                                     |
	| UCHIJAAAAAAAAA | jei               | 4.11.0.202     | jei_1.12.2-4.11.0.202.jar                   | None                                     |
	| UCHIJAAAAAAAAA | journeymap        | 1.12.2-5.5.2   | journeymap-1.12.2-5.5.2.jar                 | None                                     |
	| UCHIJAAAAAAAAA | lunatriuscore     | 1.2.0.42       | LunatriusCore-1.12.2-1.2.0.42-universal.jar | None                                     |
	| UCHIJAAAAAAAAA | schematica        | 1.8.0.169      | Schematica-1.12.2-1.8.0.169-universal.jar   | None                                     |
	| UCHIJAAAAAAAAA | stellarapi        | 1.12.2-0.4.2.9 | Stellar+API-1.12.2-0.4.2.9.jar              | None                                     |
	| UCHIJAAAAAAAAA | stellarsky        | 1.12.2-0.5.2.8 | Stellar+Sky-1.12.2-0.5.2.8.jar              | None                                     |
	| UCHIJAAAAAAAAA | vehicle           | 0.19.0         | vehicle-0.19.0-mc1.12.2.jar                 | None                                     |
	| UCHIJAAAAAAAAA | worldedit         | 6.1.8          | worldedit-forge-mc1.12-6.1.8-dist.jar       | None                                     |

	Loaded coremods (and transformers): 
CreativePatchingLoader (CreativeCore_v1.9.25_mc1.12.2.jar)
  
ItemPatchingLoader (ItemPhysic_Full_1.4.13_mc1.12.2.jar)
  com.creativemd.itemphysic.ItemTransformer
ObfuscatePlugin (Obfuscate-Mod-1.12.2.jar)
  com.mrcrayfish.obfuscate.asm.ObfuscateTransformer
	GL info: ' Vendor: 'NVIDIA Corporation' Version: '4.6.0 NVIDIA 398.36' Renderer: 'GeForce GTX 1050 Ti/PCIe/SSE2'
	Launched Version: 1.12.2-forge1.12.2-14.23.4.2705
	LWJGL: 2.9.4
	OpenGL: GeForce GTX 1050 Ti/PCIe/SSE2 GL version 4.6.0 NVIDIA 398.36, NVIDIA Corporation
	GL Caps: Using GL 1.3 multitexturing.
Using GL 1.3 texture combiners.
Using framebuffer objects because OpenGL 3.0 is supported and separate blending is supported.
Shaders are available because OpenGL 2.1 is supported.
VBOs are available because OpenGL 1.5 is supported.

	Using VBOs: No
	Is Modded: Definitely; Client brand changed to 'fml,forge'
	Type: Client (map_client.txt)
	Resource Packs: Faithful+1.12.2-rv4.zip
	Current Language: English (US)
	Profiler Position: N/A (disabled)
	CPU: 8x Intel(R) Core(TM) i7-6700 CPU @ 3.40GHz
```

**Comment by Lunatrius** [comment]
It appears to be an incompatibility with stellarsky. Can you try creating other schematic file and see if the same happens with different blocks?

**Comment by Lunatrius** [comment]
Closing due to lack of feedback.
